package.json

```diff
@@ -6,9 +6,14 @@
 		"type": "git",
 		"url": "https://github.com/misskey-dev/misskey.git"
 	},
+	"packageManager": "yarn@3.2.1",
+	"workspaces": [
+		"client",
+		"backend",
+		"sw"
```

Comment:
Are they works without `packages/`?

Reply:
Yea, `workspaces` automatically looks in the `packages` folder.

Reply:
After some more research, turns out I was entirely wrong -_-